packages/apps-config/src/ui/logos/index.ts

```diff
@@ -51,6 +51,7 @@ import chainRococoTrick from './chains/rococo-trick.svg';
 import chainShiden from './chains/shiden.png';
 import chainSkyeKiwi from './chains/skyekiwi.png';
 import chainSnakenet from './chains/snakenet.svg';
+import chainSnowbridge from './chains/snowbridge.jpeg';
```

Comment:
Ummmm... weird, see https://github.com/polkadot-js/apps/runs/6554470803?check_suite_focus=true#step:3:695



I think this could just be since we have not had jpeg before, so we don't have a loader? Although the loaders array does seem to include it, https://github.com/polkadot-js/apps/blob/master/packages/apps/webpack.base.cjs#L81-L96 - so a bit confused here.